src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -911,15 +911,18 @@ private void DynamoViewModelRequestViewOperation(ViewOperationEventArgs e)
 
         private void DynamoLoadedViewExtensionHandler(ViewLoadedParams loadedParams, IEnumerable<IViewExtension> extensions)
         {
-            foreach (var ext in extensions)
+            if (!dynamoViewModel.Model.IsServiceMode)
```

Comment:
@reddyashish This would not be triggered because Dynamo View is not initialized in WPFCLI mode or CLI mode. I suspect there are other places we launch view extensions. Will try to locate, I may revert this change

Reply:
yeah, my understanding was that these view extensions are loaded after the view is initialized. I will look too if there is any other place where it is launched.

Reply:
@reddyashish Gotcha, no wonder the DynamoViewModel initialization did not impact launch time that much